pkg/controller/controller.go

```diff
@@ -566,26 +563,9 @@ func (p *csiProvisioner) Provision(options controller.VolumeOptions) (*v1.Persis
 		return nil, fmt.Errorf("failed to strip CSI Parameters of prefixed keys: %v", err)
 	}
 
-	opts := wait.Backoff{Duration: backoffDuration, Factor: backoffFactor, Steps: backoffSteps}
-	err = wait.ExponentialBackoff(opts, func() (bool, error) {
-		ctx, cancel := context.WithTimeout(context.Background(), p.timeout)
-		defer cancel()
-		rep, err = p.csiClient.CreateVolume(ctx, &req)
-		if err == nil {
-			// CreateVolume has finished successfully
-			return true, nil
-		}
-
-		if status, ok := status.FromError(err); ok {
-			if status.Code() == codes.DeadlineExceeded {
-				// CreateVolume timed out, give it another chance to complete
-				klog.Warningf("CreateVolume timeout: %s has expired, operation will be retried", p.timeout.String())
-				return false, nil
-			}
-		}
-		// CreateVolume failed , no reason to retry, bailing from ExponentialBackoff
-		return false, err
-	})
+	ctx, cancel := context.WithTimeout(context.Background(), p.timeout)
```

Comment:
this is still the rpc connection timeout value right?  Do we want to have a separate timeout value for the CreateVolume call?  For some drivers, the provisioning operation could take minutes.

Reply:
No, the value is not right. I'd like to address it in a separate PR.

Reply:
Filled https://github.com/kubernetes-csi/external-provisioner/issues/227